pom.xml

```diff
@@ -88,8 +88,11 @@
             <artifactId>flink-test-utils_${scala.binary.version}</artifactId>
             <version>${flink.version}</version>
         </dependency>
-        <!-- zeromq -->
-
+        <dependency>
+            <groupId>org.mortbay.jetty</groupId>
+            <artifactId>jetty-util</artifactId>
```

Comment:
I know it is used in the code since I just ran into a classloader error when upgrading to 1.4.0, but shouldn't this not be needed anymore since it was transitively brought in through the [hadoop dependencies](https://flink.apache.org/news/2017/12/12/release-1.4.0.html#hadoop-free-flink-1) which are not required in 1.4.0?

Reply:
It's not needed any longer. Only `bufferSize` was imported from it. Replacing it with constants and pushing a fix up.